.github/workflows/sub_lint.yml

```diff
@@ -77,7 +77,73 @@ on:
         type: boolean
         required: false
       cpplintFilters:
-        default: -whitespace-*
+        default: >-
+          '
+          +build/deprecated,
+          +build/endif_comment,
+          +build/explicit_make_pair,
+          +build/forward_decl,
+          +build/header_guard,
+          +build/include,
+          +build/include_subdir,
```

Comment:
From what I have seen this is specific to Google's naming convention and may collide with 3rd party APIs. E.g. for QObject it's not recommended to write `#include <QtCore/QObject>` but `#include <QObject>`

Reply:
I agree, let's disable `build/include_subdir`.